providers/grammy_provider.ts

```diff
@@ -39,17 +40,23 @@ export default class GrammyProvider {
   async boot() {
     const router = await this.app.container.make('router')
     const logger = await this.app.container.make('logger')
-    const config = this.app.config.get<GrammyConfig>('grammy', this.config)
+    const config = this.app.config.get<GrammyConfig<C>>('grammy', this.config)
 
     this.bot = await this.app.container.make('grammy')
-    const { apiToken, secretToken: secret, onTimeout: timeout, timeoutMilliseconds: ms } = config
+    const {
+      apiToken,
+      secretToken: secret,
+      onTimeout: timeout,
+      timeoutMilliseconds: ms,
+      botRouteName: path,
+    } = config
 
     if (!this.initialized) {
       await this.bot.init()
       this.initialized = true
     }
 
-    router.post(apiToken, async (ctx: HttpContext) => {
+    router.post(path ? path : apiToken, async (ctx: HttpContext) => {
```

Comment:
Why don't we use shorthand like `path || apiToken` ?

Reply:
Sure we can us that.